src/_includes/layouts/home.njk

```diff
@@ -45,6 +45,10 @@
 
         <section class="container-xl mx-auto xl:px-0 sm:px-500 px-300">
           {% block content %}{{ content | safe }}{% endblock %}
+
+          <gcds-date-modified class="mb-400 font-size-caption">
```

Comment:
Same comment, can you change `font-size-caption` to `font-caption` so that it uses the caption line height as well?